Gemfile

```diff
@@ -1,27 +1,16 @@
 eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 
+# Client-side dependencies
+eval_gemfile(File.expand_path("Gemfile.client.rb", __dir__))
+# Please do not add client-side dependencies here, use Gemfile.client.rb
+
 #
 # VMDB specific gems
 #
 
 gem "rails",                           RAILS_VERSION
 gem "activerecord-deprecated_finders", "~>1.0.4",     :require => "active_record/deprecated_finders"
-
-# Client-side dependencies
-gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
-gem 'angularjs-rails', '~>1.4.3'
-gem 'angular-ui-bootstrap-rails', '~> 0.13.0'
-gem 'momentjs-rails', '~> 2.10.3'
-gem 'jquery-rails', "~>4.0.4"
-gem 'jquery-hotkeys-rails'
-gem 'codemirror-rails', "=4.2"
-gem 'lodash-rails', '~> 3.10.0'
-
-# On MS Windows run "bundle config --local build.libv8 --with-system-v8" first
```

Comment:
@djberg96 Do you remember which dependency this line was supposed to be about please? (From 984d7ae2)